tools/check/check.go

```diff
@@ -123,7 +125,8 @@ var (
 	product   = flag.String("product", "", "The AMD product name for the chip that generated the attestation report.")
 	cabundles = flag.String("product_key_path", "",
 		"Colon-separated paths to CA bundles for the AMD product. Must be in PEM format, ASK, then ARK certificates. If unset, uses embedded root certificates.")
-	verbose = flag.Bool("v", false, "Enable verbose logging.")
+	verbose     = flag.Bool("v", false, "Enable verbose logging.")
+	testKdsFile = flag.String("kdsdatabase", "", "Path to a fakekds.Certificates binary cache of AMD KDS")
```

Comment:
I'm not sure whether a test-only flag is appropriate here since check.go is not test code? I guess even if we keep it here we should explicitly mention that this flag is for test only.

Reply:
check.go is not test code, but the tool needs to be tested, so it needs some testonly flags to accommodate non-production environments.